src/config/types.ts

```diff
@@ -197,6 +203,13 @@ export type PricedBalance = Balance &
     fiatCode?: string;
     fiatTotal?: BigNumber | null;
     displayName?: string;
+    token: {
```

Comment:
Considering the `Balance` type currently is:
```
export type Balance =
  | NativeBalance
  | ClassicBalance
  | SorobanBalance
  | LiquidityPoolBalance;
```
Overwriting the `token` property here would break for `LiquidityPoolBalance` since they don't have a token property associated to it.

@Brunonascdev what do you think? Would it be possible to not overwrite `token` here and handle it differently?